packages/card/src/react/__specs__/storyshots.spec.tsx

```diff
@@ -8,5 +8,6 @@ const createNodeMock = () => document.createElement('div')
 initStoryshots({
   configPath: path.resolve(__dirname, '../../../.storybook'),
   framework: 'react',
+  // @ts-ignore
```

Comment:
This is the thing I feel the worst about.

The answer in other packages has been to remove this line. Here, it broke Shiitake in the storyshots.

Putting it back, the typings for the package don't even indicate that this is available. 

Perhaps there are new typings?  I haven't looked yet.  Perhaps we could patch/extend the typings locally. Haven't tried that yet either.

Reply:
let's make an issue and follow up in a focused pr

Reply:
carded https://github.com/pluralsight/design-system/issues/1355